Contents.m

```diff
@@ -0,0 +1,8 @@
+% MATLAB-bindings for coupling library preCICE
```

Comment:
Is this the official name of the package?

Reply:
I think, this name should be consistent with whatever name we use in the [`README.md`](https://github.com/precice/matlab-bindings#matlab-bindings). Also keep our naming discussion https://github.com/precice/fenics-adapter/issues/85 in mind. Is there a straightforward choice?

I think our `README.md` should use the full name. `Contents.m` can use the long name or a short name (similar to "Python language bindings for the C++ library preCICE" vs. [`pyprecice`](https://pypi.org/project/pyprecice/)), depending on what the MATLAB community usually does here. What do other packages do?

Reply:
MATLAB software comes with several [toolboxes](https://de.mathworks.com/products.html) and every toolbox has its own Contents.m file. For example, for the Communications Toolbox the file is located at: `R2018b/./toolbox/comm/comm/Contents.m` and the first line is:
```
% Communications Toolbox
% Version 7.0 (R2018b) 24-May-2018
```
* It can be deduced that in terms of the preCICE naming, the first line should be the full adapter name rather than the package. 
* The second line states the version, the MATLAB version with which this package is shipped, and the date of development.

I propose we follow the same nomenclature.

Reply:
Sounds good. Meaning: We will use the full name here.

Let's continue the discussion on `(R2018b)` below.